From 139b280bb61305bc21cc8ba74309ca5c3b9e8840 Mon Sep 17 00:00:00 2001
From: Ben Hutchings <ben@decadent.org.uk>
Date: Sat, 7 Apr 2012 02:40:01 +0100
Subject: [PATCH 2/2] ARM: Fix missing #include for call to soft_restart() in
 plat-iop/restart.c

This is for the stable 3.3.y branch only.  The bug was fixed in
mainline as part of commit 9f97da78bf018206fb623cd351d454af2f105fe0
('Disintegrate asm/system.h for ARM').

Signed-off-by: Ben Hutchings <ben@decadent.org.uk>
---
 arch/arm/plat-iop/restart.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/plat-iop/restart.c b/arch/arm/plat-iop/restart.c
index 6a85a0c..a6ee3b6 100644
--- a/arch/arm/plat-iop/restart.c
+++ b/arch/arm/plat-iop/restart.c
@@ -8,6 +8,7 @@
  * published by the Free Software Foundation.
  */
 #include <asm/hardware/iop3xx.h>
+#include <asm/system.h>
 #include <mach/hardware.h>
 
 void iop3xx_restart(char mode, const char *cmd)
-- 
1.7.9.5

